.rubocop_todo.yml

```diff
@@ -36,6 +38,7 @@ Metrics/MethodLength:
 Metrics/PerceivedComplexity:
   Exclude:
     - 'lib/active_encode/engine_adapters/ffmpeg_adapter.rb'
+    - 'lib/active_encode/engine_adapters/media_convert_adapter.rb'
     - 'lib/file_locator.rb'
```

Comment:
Why does update of ruby/rails versions require excluding more files from these metrics?

Reply:
The default metrics for these cops were reduced in the upgrade to rubocop 1.

Reply:
would you prefer to set the metrics to a custom higher level (matching what they were before), vs excluding the files from metric analysis altogether? is that possible?

Reply:
Good point.  I'll look into that once the question whether we continue using bixby (via upgrading) or not is resolved.

Reply:
Removing this worked after upgrading to bixby 4.0.0.  I think I raised the threshold for this cop in bixby based upon the threshold used in hyrax.